pic-configure

```diff
@@ -55,7 +54,7 @@ get_backend_flags()
     elif [ "${backend_cfg[0]}" == "omp2b" ] ; then
         result+=" -DALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLE=ON"
         if [ $num_options -eq 2 ] ; then
-            result+=" -DCMAKE_CXX_FLAGS=\"-march=${backend_cfg[1]} -mtune=${backend_cfg[1]}\""
+            result+=" -DCMAKE_CXX_FLAGS=\"$CXXFLAGS -march=${backend_cfg[1]} -mtune=${backend_cfg[1]}\""
```

Comment:
note: the order is important here.
you want to prefer the cmd line arg over the environment setting. the latter argument in the compiler arg list is preferred, so this line is correct.